tests/plugins/preview/manual/previewcdn.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: 4.16.2, 4444, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, print, font, colorbutton, format, clipboard, pagebreak, toolbar, floatingspace, link, image2
```

Comment:
This test and PR is about `preview` plugin so I;m not sure if `print` is needed :thinking:

Reply:
My bad. I left it after the first PR version which included a problem with print and preview.